DOCS.md

```diff
@@ -16,8 +16,8 @@ The following parameters are used to configure this plugin:
 * *optional* `template` - Kubernetes template (like the [deployment object](http://kubernetes.io/docs/user-guide/deployments/)) (defaults to `.kube.yml`)
 * *optional* `secret_template` - Kubernetes template for the [secret object](http://kubernetes.io/docs/user-guide/secrets/) (defaults to `.kube.sec.yml`)
 * `vars` - variables to use in `template`
-* `secrets` - variables to use in `secret_template`. These are base64 encoded by the plugin.
-* `secrets_base64` - variables to use in `secret_template`. These should already be base64 encoded; the plugin will not do so.
+* `kube_secrets` - variables to use in `secret_template`. These are base64 encoded by the plugin.
```

Comment:
This rename from `secrets` to `kube_secrets` has to happen because `secrets` is now used by drone.